src/pages/trainer/dashboard/page.tsx

```diff
@@ -12,6 +12,7 @@ export class DashboardPage extends React.Component<{}, {}> {
            <br/>
            <Link to="/trainers/evaluation">Go to student evaluation</Link>
            <Link to="/trainers/">Go back to training</Link>
+           <Link to="/trainers/training/edit/1">Edit training content</Link>
```

Comment:
Yes, this is only to fetch training with Id 1 from API. As you said, it's only temporal